lib/active_job/queue_adapters/delayed_job_adapter.rb

```diff
@@ -0,0 +1,61 @@
+module ActiveJob
+  module QueueAdapters
+    # Explicitly remove the implementation existing in older rails'.
+    remove_const(:DelayedJobAdapter) if defined?(:DelayedJobAdapter)
```

Comment:
`defined?(:DelayedJobAdapter)` will always be true since a symbol is passed in.

```suggestion
    remove_const(:DelayedJobAdapter) if const_defined?(:DelayedJobAdapter)
```

Reply:
Thanks @Earlopain